src/views/Sign-Up.js

```diff
@@ -0,0 +1,175 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import {
+  handleRegistration,
+  handleInputChange,
+  handleBlur
+} from "../redux/action-creators/registration.action-creator";
+import TextInput from "../components/common/Inputs/TextInput";
+import checkField from "../utils/check-field";
+import FormButton from "../components/common/Buttons/FormButton";
+
+const mapStateToProps = state => {
+  const { registration } = state;
+  return registration;
+};
+
+export class SignUp extends Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
User arrow functions which don't require binding.

Reply:
what do you mean by that? I'm confused...

Reply:
@espoirMur, I think using Arrow functions means to define your class methods with arrow functions which don't require to be bound with `this` context of the class.

Reply:
Is there any technical problem with that way?

Reply:
There is no problem, but it gives you much work binding all the class methods which can accidentally result in forgetting to bind some of them. Are there any concerns/problems from @espoirMur about defining class methods with arrow functions?

Reply:
Yes, there is a problem! 
I have tried your suggestion but it no longer working on my side...
Maybe you can help...

Reply:
@abayo-luc, Thanks for this suggestion. I'm done with the implementation...